azurelinuxagent/common/conf.py

```diff
@@ -146,7 +146,8 @@ def load_conf_from_file(conf_file_path, conf=__conf__):
     "Debug.CgroupDisableOnQuotaCheckFailure": True,
     "Debug.EnableAgentMemoryUsageCheck": False,
     "Debug.EnableFastTrack": True,
-    "Debug.EnableGAVersioning": True
+    "Debug.EnableGAVersioning": True,
+    "Debug.EnableExtensionPolicy": True
```

Comment:
Let's leave this as False since e2e functionality isn't ready.

When testing, we should change EnableExtensionPolicy to True

Reply:
Updated